lib/osf-components/addon/components/osf-layout/registries-side-nav/label/component.ts

```diff
@@ -0,0 +1,38 @@
+import { tagName } from '@ember-decorators/component';
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import { assert } from '@ember/debug';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import defaultTo from 'ember-osf-web/utils/default-to';
+
+import styles from './styles';
+import template from './template';
+
+@tagName('')
+@layout(template, styles)
+export default class Label extends Component {
+    // Required parameters
+    label!: string;
+
+    // Optional parameters
+    count?: number;
+
+    // Private properties
+    isCollapsed: boolean = defaultTo(this.isCollapsed, false);
+
+    @computed('count')
+    get hasCount() {
+        return typeof this.count === 'number';
+    }
+
+    @computed('count', 'isCollapsed')
+    get showCount() {
+        return this.hasCount && !this.isCollapsed;
+    }
+
+    didReceiveAttrs() {
+        assert('OsfLayout::RegistriesSideNave::Label: ' +
```

Comment:
Same thing down here.  One line to make it a bit more readable.  Also should be Nav instead of Nave.
```suggestion
        assert(
            'OsfLayout::RegistriesSideNav::Label: @label is required for this component to render',
            Boolean(this.label),
        );
```